typesafety/test_primitives/test_tracing/test_collect_traces.yml

```diff
@@ -0,0 +1,35 @@
+- case: collect_traces_context_manager_return_type_one
+  disable_cache: true
+  main: |
+    from returns.primitives.tracing import collect_traces
+
+    reveal_type(collect_traces)  # N: Revealed type is 'Overload(def () -> typing.ContextManager[None], def [Function <: def (*Any, **Any) -> Any] (function: Function`-1) -> Function`-1)'
+
+- case: collect_traces_context_manager_return_type_two
+  disable_cache: true
+  main: |
+    from returns.primitives.tracing import collect_traces
+
+    reveal_type(collect_traces())  # N: Revealed type is 'typing.ContextManager[None]'
+
+- case: collect_traces_decorated_function_return_type
```

Comment:
We can safely replace this test with `with collect_traces()` one.

Reply:
I don't know if I have did right:
```python
with reveal_type(collect_traces()):  # N: Revealed type is 'typing.ContextManager[None]'
    pass
```